R/calculate_esi.R

```diff
@@ -0,0 +1,38 @@
+#' Calculate the Earth Similarity Index (ESI)
+#'
+#' @details
+#' The function calculates the ESI based on stellar flux and planetary radius.
+#'
+#' ESI (Earth Simillarity Index) is a characterization of how simmilar a planetary-mass object
+#' or natural satelite is to earth. It was designed to be a scale from zero to one,
+#' with Earth having a value of 1.
+#'
+#'
+#' @references
+#' Schulze-Makuch, D., Méndez, A., Fairén, A. G., von Paris, P., Turse, C., Boyer, G.,
+#' Davila, A. F., Resendes de Sousa António, M., Irwin, L. N., and Catling, D. (2011)
+#' A Two-Tiered Approach to Assess the Habitability of Exoplanets. Astrobiology 11(10): 1041-1052.
+#'
+#' @param pl_rade Numeric. Planetary radius in Earth radii.
+#' @param pl_insol Optional numeric. Stellar flux in Earth units.
```

Comment:
nitpick: Lets keep similar order of information in documentation.
```suggestion
#' @param pl_insol Numeric. Stellar flux in Earth units. Optional.
```